Sources/Extensions/UIView+SuperviewOffset.swift

```diff
@@ -30,6 +34,15 @@ enum Offset {
 }
 
 extension UIView {
+
+    fileprivate var insets: UIEdgeInsets {
```

Comment:
Looks like code duplication: Assistive button has similar implementation

Reply:
So that's a little hack to implement safe area support inside this extension file more confirmable.